hushline/routes.py

```diff
@@ -72,28 +72,31 @@ def inbox() -> Response | str:
         # Redirect if not logged in
         if "user_id" not in session:
             flash("Please log in to access your inbox.")
-            return redirect(url_for("login"))
+            return redirect(url_for("login")), 302
```

Comment:
The correct way to do this would be `redirect(url_for('login'),  code=302)` but since 302 is the default, this is unnecessary.

Reply:
👍